doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,129 @@
+.. _ug_nrf70_developing_raw_tx_operation:
+
+Raw packet transmission
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nrf70 series device supports the transmission of raw packets. Raw packets are packets that are not touched by the 802.11 MAC layer during transmission. The Application should provide an 802.11 conformant packet with transmit parameters which the 802.11 MAC layer will utilize to transmit packets.
+
+
+.. _ug_nrf70_developing_enabling_raw_packet_transmit_feature:
+
+Enabling raw packet transmit feature
+************************************
+
+The raw packet transmit feature for the nrf70 series device is placed under a configuration flag ``CONFIG_NRF700X_RAW_DATA_TX``. The configuration setting can be enabled by enabling the configuration in the ``prj.conf`` of the application attempting to send raw packets.
+
+.. _ug_nrf70_developing_raw_tx_operating_modes:
+
+Raw packet operating modes
+**************************
+
+The nrf70 series device supports raw packet transmission when TX-Injection mode is enabled. A net_mgmt API ``NET_REQUEST_WIFI_MODE`` has been provided to set TX-Injection mode of operation.
+
+The nrf70 series device supports raw packet transmission in the below station operating modes:
+
+* **Station idle mode:** The device is operating as an 802.11 station device and is in idle state (not connected to AP).
+* **Station connected mode:** The device is operating as an 802.11 station device and is in connected state.
+
+To enable raw packet transmission, the nrf70 device mode of operation has to be configured to "Station + TX-Injector mode" using the ``NET_REQUEST_WIFI_MODE`` net_mgmt API. When the raw tx mode is enabled, a channel for raw packet transmission also has to be configured mandatorily. This channel will be utilized for raw packet transmission when the station is in disconnected state. When the station is in connected state, the station connected channel will be used for raw packet transmission. Channel configuration cannot be done when station is in connected state. The net_mgmt API ``NET_REQUEST_WIFI_CHANNEL`` can be used to configure the channel of operation.
```

Comment:
> To enable raw packet transmission, the nrf70 device mode of operation has to be configured to "Station + TX-Injector mode" using the ``NET_REQUEST_WIFI_MODE`` net_mgmt API.

 This contradicts above statement of two modes.

Reply:
I would rather not use the "raw TX mode" term. Let's stick to fewer terms
- the mode is "TX Injection"
- the operation is "raw packet transmission"

Reply:
MAybe simplify the description here. The key message is that when the Station is connected, the raw injection mode will only transmit raw packets on the channel the Station is connected/operating on.

An additional message is the NET mgmt API that can be used to set the tx injection mode channel.

Reply:
"Station + TX-Injector mode"
We probably need links where these modes of operation are defined.

Reply:
done